Velaptor/NativeInterop/OpenGL/IGLInvoker.cs

```diff
@@ -452,7 +452,7 @@ internal interface IGLInvoker : IDisposable
         /// </param>
         /// <param name="normalized">
         ///     For glVertexAttribPointer, specifies whether or not fixed-point data values should be
```

Comment:
Should the underlined text be spelled without spaces? If so, should it have quotes or be underlined?
```suggestion
        ///     For glVertexAttribPointer, specifies whether or not fixed-point data values should be
```